src/wasm/wasm-emscripten.cpp

```diff
@@ -32,7 +33,10 @@ namespace wasm {
 cashew::IString EMSCRIPTEN_ASM_CONST("emscripten_asm_const");
 cashew::IString EM_JS_PREFIX("__em_js__");
 
-static constexpr const char* dummyFunction = "__wasm_nullptr";
+static Name STACK_SAVE("__stackSave"),
```

Comment:
This changes `stackSave` to `__stackSave`? But `stackRestore` still doesn't start with the dunder? I'm confused.

Reply:
oops, good catch.